### PR TITLE
When kubectl exec dig, the dns perf client selector podname is not  correct

### PR DESCRIPTION
**Pull request by yuzhiquan**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:
when kubectl exec dig, the dns perf client selector podname is not  correct
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: dns/py/runner.py [hunk 1]
```diff
@@ -444,13 +439,22 @@ def _wait_for_status(self, active):
       time.sleep(1)
 
     if active:
+      break_loop = False
       while True:
-        code, out, err = self._kubectl(
+        client_pods = self._get_pod('app=dns-perf-client')
+        if client_pods == 0:
```
Comment (prameshj) [inline dns/py/runner.py:445]: len(client_pods)?

File: dns/py/runner.py [hunk 2]
```diff
@@ -459,6 +463,16 @@ def _wait_for_status(self, active):
 
       _log.info('DNS is up')
 
+  def _get_pod(self, label, output='custom-columns=:metadata.name'):
+    code, client_pods, err = self._kubectl(None, 'get', 'pods', '-l', label, '--no-headers', '-o',
```
Comment (prameshj) [inline dns/py/runner.py:467]: The label selector looks identical to what exists in the code already. "app=dns-perf-client" What is this fixing?

File: dns/py/runner.py [hunk 3]
```diff
@@ -444,13 +439,22 @@ def _wait_for_status(self, active):
...
+      break_loop = False
       while True:
-        code, out, err = self._kubectl(
+        client_pods = self._get_pod('app=dns-perf-client')
+        if client_pods == 0:
+          continue
+        for podname in client_pods:
+          code, out, err = self._kubectl(
             None,
-            'exec', _client_podname, '--',
```
Comment (prameshj) [inline dns/py/runner.py:450]: I see the issue now. This part was harcoding client podname as "dns-perf-client". Can you also remove the definition in line 34?

File: dns/py/runner.py (hunk 2 above)
Comment (prameshj) [reply dns/py/runner.py:467]: ignore this.. I found the issue in one exec call which was hardcoding client name.

File: dns/py/runner.py (hunk 1 above)
Comment (yuzhiquan) [reply dns/py/runner.py]: Ops, fix it.

File: dns/py/runner.py (hunk 3 above)
Comment (yuzhiquan) [reply dns/py/runner.py:450]: Sure, and define "_client_label".

**Comment by yuzhiquan** [comment]
Gently ping @prameshj

**Pull request merged**